packages/flutter/test/widgets/image_test.dart

```diff
@@ -353,14 +353,14 @@ void main() {
         child: image,
       ),
     );
-    expect(imageStreamCompleter.listeners.length, 1);
+    expect(imageStreamCompleter.listeners.length, 2);
     await tester.pumpWidget(
       new TickerMode(
         enabled: false,
         child: image,
       ),
     );
-    expect(imageStreamCompleter.listeners.length, 0);
+    expect(imageStreamCompleter.listeners.length, 2);
   });
```

Comment:
add a test to verify that the maximum size is respected

Reply:
should be covered here https://github.com/flutter/flutter/pull/18135/files#diff-29dd8c7005353481beef7d2ab47bd9aeR49